geps/gep-1619.md

```diff
@@ -210,18 +666,157 @@ TBD - See [open question](#open-questions) about session affinity API
 - Should we include both session persistence and session affinity API configurations in this GEP or just focus on session persistence?
 - Should both session persistence and session affinity be designed as common load balancing policy API? Should they be configured as separately in the API?
 - What happens when session persistence is broken because the backend is not up or healthy? If that's an error case, how should that be handled? Should the API dictate the http error code? Or should the API dictate fall back behavior?
```

Comment:
I think this GEP (if not the API) should dictate the fallback behavior - and that behavior should be to fall back to the applicable LB policy (which was overridden by session persistence).

It is also preferable to specify the HTTP error code (in case of an error occurring even after following the fallback behavior)

Reply:
Noted, I agree as that seems the the most obvious and simple solution. I'll keep this in mind, but tackle this in another PR to keep this one lower scoped.